doc/user_manual_en.md

```diff
@@ -0,0 +1,111 @@
+* _Emoticon size:_ You can change the size of the emoticons here.
+* _Style:_ Changes the appearance of qTox.<!-- TODO: possible to add own styles? -->
+* _Theme color:_ Changes the colors qTox uses.<!-- TODO: possible to add own color schemes? -->
+* _Timestamp format:_ Change the format in which qTox displays message timestamps.
+* _Date format:_ Same as above for the date.
+
+#### Connection Settings
+* _Enable IPv6 (recommended):_ If enabled, qTox will use IPv4 and IPv6 protocols, whichever is available. If disabled, qTox will only use IPv4.
+* _Enable UDP (recommended):_ If enabled, qTox will use TCP and UDP protocols. If disabled, qTox will only use TCP.<!-- TODO: correct? what advantages/disadvantages does this have? -->
+* _Proxy type:_ If you want to use a proxy, set the type here. "None" disables the proxy.
+* _Address:_ If you use a proxy, enter the address here.
+* _Port:_ If you use a proxy, enter the port here.
+* _Reconnect:_ Reconnect to the tox network, e.g. if you changed the proxy settings.
+
+### Privacy
+
+* _Send typing notifications:_ If enabled, notify your chat partner when you are currently typing.
+* _Keep chat history:_ If enabled, qTox will save your sent and recieved messages. Encrypt your profile, if you want to encrypt the chat history.
+
+#### NoSpam
+
+NoSpam is a feature of Tox to prevent someone spamming you with friend requests. If you get spammed, enter or generate a new NoSpam value, this will alter your Tox ID. You don't need to tell your existing contacts your new Tox ID, but you have to tell new contacts your new Tox ID. Your Tox ID can be found ??? <!-- TODO: link to page describing your profile-->
+
+### Audio/Video
+#### Audio Settings
+* _Playback device:_ Select the device qTox should use for all audio output (notifications, calls,..).
+* _Playback:_ Here you can adjust the playback volume to your needs.
+* _Capture device:_ Select the device qTox should use for audio input in calls.
+* _Microphone:_ Set the input volume of your microphone with this slider. When you are talking normaly, the the display should be in the green range.<!-- TODO: better wording-->
+* _Filter audio_ If enabled, qTox will try to remove noise and echo from your audio input.
+
+#### Video Settings
+* _Video device:_ Select the video device qTox should use for video calls. "None" will show a dummy picture to your chat partner. "Desktop" will stream the content of your screen.
+* _Resolution:_ You can select from the available resolutions and framerates here. Higher resolutions provide more quality, but if the bandwidth of your connection is low, the video may get choppy.
+
+  If you set up everything correctly, you should see the preview of your video device in the box.
+
+* _Rescan devices_ Use this button to search for newly attached devices, e.g. you plugged in a webcam.
+
+### Advanced
+* _Make Tox portable:_ ???<!-- TODO: What does this do exactly?-->
```

Comment:
Saves/loads user data to/from working directory, instead of using `~/.config/tox/`